spec/controllers/users_controller_spec.rb

```diff
@@ -203,6 +203,13 @@
         expect(Workers::Mail::ConfirmEmail).to receive(:perform_async).with(@user.id).once
         put(:update, :id => @user.id, :user => { :email => "my@newemail.com"})
       end
+
+      it "saves unconfirmed_email when podmail is disabled" do
+        AppConfig.mail.enable = false
+        put(:update, :id => @user.id, :user => { :email => "my@newemail.com"})
```

Comment:
For bigger pull requests it's a good idea to keep unrelated changes separate. This makes the changes more readable and makes the review easier.

In this case it's just a small PR, so do whatever you like. :)